Mathlib/Data/Sum/Basic.lean

```diff
@@ -111,6 +111,18 @@ theorem update_inr_apply_inr [DecidableEq β] [DecidableEq (α ⊕ β)] {f : α
     {x : γ} : update f (inr i) x (inr j) = update (f ∘ inr) i x j := by
   rw [← update_inr_comp_inr, Function.comp_apply]
 
+@[simp]
+theorem update_inl_inl_apply {γ : α ⊕ β → Type*} [DecidableEq α] [DecidableEq (α ⊕ β)]
```

Comment:
On second thoughts, I think
```suggestion
theorem update_inl_apply_inl {γ : α ⊕ β → Type*} [DecidableEq α] [DecidableEq (α ⊕ β)]
```
is the better name, since there are no applications after the second `inl`. Same below.